content/en/docs/components/pipelines/sdk/output-viewer.md

```diff
@@ -55,10 +54,173 @@ All screenshots and code snippets on this page come from a
 sample pipeline that you can run directly from the Kubeflow Pipelines UI.
 See the [sample description and links below](#example-source).
 
-## Writing out metadata for the output viewers
+<a id="v2-visualization"></a>
+## v2 SDK: Use SDK visualization APIs
 
-The pipeline component must write a JSON file specifying metadata for the
-output viewer(s) that you want to use for visualizing the results. The
+For KFP [SDK v2 and v2 compatible mode](/docs/components/pipelines/sdk/v2/), you can use 
+convenient SDK APIs and system artifact types for metrics visualization. Currently KFP
+supports ROC Curve, Confusion Matrix and Scalar Metrics formats. Full pipeline example
+of all metrics visualizations can be found in [metrics_visualization_v2.py](https://github.com/kubeflow/pipelines/blob/master/samples/test/metrics_visualization_v2.py). For a usage guide of each metric visualization output,
+refer to sections below:
```

Comment:
I'd suggest adding a section for requirements, users can only use the feature after next KFP release.

Also, therefore we cannot merge this PR yet. It might make sense if we set up a branch specifically for KFP v2 compatible. Then we can keep adding PRs to the branch, and merge all to master branch in one batch.

Reply:
Sounds good, I have created a `kubeflow:kfp-v2-compatible` branch for storing the documentation for items pending KFP release.

I also added a requirement section for being able to use V2 metrics visualization.